bundles/org.openhab.binding.bluetooth.bluez/src/main/resources/OH-INF/i18n/bluetooth.properties

```diff
@@ -13,3 +13,5 @@ thing-type.config.bluetooth.bluez.inactiveDeviceCleanupInterval.label = Device C
 thing-type.config.bluetooth.bluez.inactiveDeviceCleanupInterval.description = How often device cleanup is performed
 thing-type.config.bluetooth.bluez.inactiveDeviceCleanupThreshold.label = Device Cleanup Threshold
 thing-type.config.bluetooth.bluez.inactiveDeviceCleanupThreshold.description = Timespan a device can remain radio silent before it is eligible for cleanup
+thing-type.config.bluetooth.bluez.lazyScan.label = Lazy Scan
+thing-type.config.bluetooth.bluez.lazyScan.description = Disables re-querying of GATT services from already known devices when scanning
```

Comment:
Can you also update https://github.com/openhab/openhab-addons/blob/main/bundles/org.openhab.binding.bluetooth.bluez/src/main/resources/OH-INF/thing/bluez.xml and add it to the README?